test/database/sqlite/schema_utils.py

```diff
@@ -500,6 +500,9 @@ def _add_with_references(filled: T, service: BaseService):
                 it.add_energized_loop(filled)
                 service.add(it)
 
+        if isinstance(filled, LvFeeder):
```

Comment:
question: Does this need to add references to the feeder if it is a `LvFeeder`? I am not sure why this is not failing somewhere, perhaps I am missing something, perhaps there is a missing test.

Reply:
`_add_with_references` is only used in `*_services_of`, which are only used in database tests. `filled` never has runtime fields (e.g. `Feeder().current_equipment`).